crew

```diff
@@ -27,6 +27,9 @@ else
   CREW_NPROC = ENV["CREW_NPROC"]
 end
 
+# Set CREW_NOT_COMPRESS from environment variable
+CREW_NOT_COMPRESS = ! File.exist?("#{CREW_PREFIX}/bin/compressdoc") || ENV["CREW_NOT_COMPRESS"]
```

Comment:
I know I'm probably overthinking this and should just get back to my day job but why don't we just install the `compressdoc` package if the command is not found?  Then you don't need any of this logic.  Also, if users are unaware that they need to install `compressdoc`, then we potentially get a bunch of packages that are installed that need to be reinstalled so the manpages are compressed.  If this is the case, then we would need a `crew compress` command to change all installed packages in one process.  Do you understand what I'm getting at here?